Fakebook.Profile/Fakebook.Profile.Domain/Fakebook.Profile.Domain.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
see above

Reply:
The project target version was changed to ```net5.0```.